packages/core/src/App/AppContent.tsx

```diff
@@ -37,16 +37,29 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
                 rudderstackKey: process.env.RUDDERSTACK_KEY,
             };
             Analytics.initialise(config);
+            const ppc_campaign_cookies =
+                Cookies.getJSON('utm_data') === 'null'
+                    ? {
+                          utm_source: 'no source',
+                          utm_medium: 'mo medium',
```

Comment:
```suggestion
                          utm_medium: 'no medium',
```
is it intended or a typo? :)

Reply:
Misspells isn't important for analytics, but let me resolve it if it cuts engineer's eyes :)